conceptnet5/readers/emoji.py

```diff
@@ -8,7 +8,7 @@
 REL = '/r/SymbolOf'
 DATASET = '/d/emoji'
 LICENSE = Licenses.cc_attribution
-SOURCE = [{'contributor': '/s/resource/unicode/cldr/31'}]
+SOURCE = [{'contributor': '/s/resource/unicode/cldr/32.0.1'}]
```

Comment:
It doesn't really matter, but I'd just leave this as "/s/resource/unicode/cldr/32". If we were using the sources for debugging, the micro version wouldn't make a meaningful distinction in the data we need to track down.

Reply:
...oh, given that this would involve changing test data again, I guess just leaving it would be fine.